packages/autorest.typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/typescript",
-  "version": "6.0.15",
+  "version": "6.0.17",
```

Comment:
It's caused by failing to bump the version in an earlier release https://github.com/Azure/autorest.typescript/commit/71273b38e5d262719e8d807c952ae3df925f4d29

Reply:
I may not get the point, why not 6.0.16?

Reply:
In the commit id I comment here, it added the change log but didn't bump the version in package.json.  cause them to be inconsistent, we don't realize this issue in our last release.